Changes the default verbosityRestore to be a more default value

aarondandy opened the pull request:
This proposed change reduces the level of noise in the output from `dotnet restore` as outlined in #12297 . I am unable to run the build or tests on my machine so I hope this does not conflict with any possible regression tests 🤞

bishal-pdMSFT commented:
Thanks @aarondandy for the PR. I do not know of any reason why it should be detailed verbosity, and hence this looks good to me. 
You will need to do two changes:
1. Increment the task patch version.
2. Make same changes in task.loc.json file as well. Ideally build would generate this file changes, but if you are not able to build locally then you can edit the file.

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/microsoft/azure-pipelines-tasks?pullRequest=12298) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/azure-pipelines-tasks?pullRequest=12298) before we can accept your contribution.<br/><br/>:x: aarondandy [sign now](https://cla.opensource.microsoft.com/microsoft/azure-pipelines-tasks?pullRequest=12298)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/microsoft/azure-pipelines-tasks?pullRequest=12298) it.</sub>

aarondandy commented:
I fixed the versions and duplicate files up as you requested. Also, I switched it from Normal to Minimal as that is the default for the `dotnet` tool according to both the docs and my testing.

Docs: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore#options

bishal-pdMSFT commented:
@aarondandy this PR needs to be synced base branch. Please do so and I will then merge it.

aarondandy commented:
@bishal-pdMSFT Can you double check that the patch increment is still correct? I am not familiar with how you version things in these PRs.

aarondandy commented:
I'm going to stop updating this PR to be up to date with master. Let me know before hand if you want to attempt merging this again and I can fix it up then. If you think this is not likely to be merged I can just close it.
